content/docs/client-options.md

```diff
@@ -26,10 +26,11 @@ third party clients.
 These clients are compatible with our [staging endpoint for ACME v2](https://community.letsencrypt.org/t/staging-endpoint-for-acme-v2/49605).
 
 - [ACME4J](https://github.com/shred/acme4j) (acme4j >= 2.0)
-- [GetSSL](https://github.com/srvrco/getssl/tree/APIv2) (`APIv2` branch)
```

Comment:
hate to nitpick but could you leave the order of the existing ACMEv2 clients unchanged and add Hiawatha at the end? The convention we've used so far has been to add new clients at the end instead of in alphabetical order. Thanks!